pkg/solidity-utils/contracts/helpers/ArrayHelpers.sol

```diff
@@ -4,7 +4,14 @@ pragma solidity ^0.8.24;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-/// @notice Library of helper functions used to convert between fixed size and dynamic arrays of various types.
+/**
+ * @notice Library of helper functions to convert fixed-sized array types to dynamic arrays.
+ * @dev For instance, the `buildTokenConfig` functions construct a `TokenConfig` from its component parts.
+ * The first argument is a dynamic array of `IERC20`, but it's convenient to start with a fixed array of strings or
+ * addresses. `[address(dai), address(usdc)].toMemoryArray().asIERC20()` casts to the correct type.
+ *
+ * This library is intended for testing purposes; it should not be used in the main codebase.
```

Comment:
If this is only for testing, maybe we should move it to the test folder? (Maybe on another PR)

Reply:
Will consider, yes. Out of scope for this PR for sure.